modules/fedimint-mint-client/src/backup/recovery.rs

```diff
@@ -246,20 +247,32 @@ impl MintRestoreInProgressState {
         decoders: ModuleDecoderRegistry,
         secret: DerivableSecret,
     ) -> Self {
+        const PROGRESS_SNAPSHOT_BLOCKS: u64 = 50;
```

Comment:
Hard to pick a good value here. On idle federations this will be quite small, on very busy ones it could be a lot.

Recovery refactoring should render this irrelevant, so I'm siding with 'go fast' for now.

Reply:
This constant was picked by multiplying previous (implicit) `1` by a logarithm of "recovery takes hours" reported by some users.